lib/modules/auth/login/login_screen.dart

```diff
@@ -1,232 +1,210 @@
-import 'package:conditional_builder/conditional_builder.dart';
 import 'package:flutter/material.dart';
+
```

Comment:
format file

Reply:
done

Reply:
format file

Reply:
i did now and there is no change in the file!